minigalaxy/window/gametile.py

```diff
@@ -69,11 +69,23 @@ def on_button_click(self, widget) -> None:
 
     @Gtk.Template.Callback("on_menu_button_uninstall_clicked")
     def on_menu_button_uninstall(self, widget):
-        self.menu_button.hide()
-        self.button.set_sensitive(False)
-        self.button.set_label(_("uninstalling.."))
-        uninstall_thread = threading.Thread(target=self.__uninstall_game)
-        uninstall_thread.start()
+        message_dialog = Gtk.MessageDialog(parent=self.parent,
+                                           flags=Gtk.DialogFlags.MODAL,
+                                           message_type=Gtk.MessageType.WARNING,
+                                           buttons=Gtk.ButtonsType.OK_CANCEL,
+                                           message_format="Are you sure to uninstall this game ?")
```

Comment:
Could you replace this game with the name of the game?

Reply:
Could you also make the string translatable? I've added instructions on how to do that here: https://github.com/sharkwouter/minigalaxy/wiki/Developer-information#making-strings-translatable